.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules
-package-lock.json
```

Comment:
So this project can both be used as a lib and as a standalone server. When installed as a lib (ie as part of ZJS2MQTT), the package-lock file will be ignored.

Reply:
If we ignore `package-lock.json` from git we basically allow to unintentionally bump the dependencies during `npm i` on github actions and contributors machines. This can lead to failed tests/builds. 
In the end this file **won't** be published to [npm by default](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json), so it won't be a problem for the lib consumers.

Reply:
We should publish it to npm, for when it's installed as a CLI. 

When an npm package is installed as a dependency, the package-lock file is already ignored anyway.

Reply:
You can publish `npm-shrinkwrap.json` only. I will add the proper command to the github action.

Reply:
Sorry to jump in, but when I install this from npm (`npm install @zwave-js/server`), it's currently pulling in zwave-js 6.0.0-beta.3 as a dependency. Is that what this conversation is about? I don't think I should have that version as a dependency yet...